tests/api/dto/test_grundsteuer_input_eigentuemer.py

```diff
@@ -100,11 +100,11 @@ def test_if_all_persons_have_test_tax_id_number_then_raise_no_error(self):
                           steuer_id="03352417692", anteil=Anteil(zaehler="1", nenner="2"))]
         Eigentuemer.parse_obj({"person": persons})
 
-    def test_if_all_persons_have_test_tax_id_number_not_special_testmerker_then_raise_error(self):
+    def test_if_all_persons_have_test_tax_id_number_or_none_then_raise_no_error(self):
```

Comment:
Yes, I am fine with that. I think the old test was some legacy test that we probably should have adapted earlier on. Thanks